VotingApplication/VotingApplication.Web.Api/Views/Manage/Index.cshtml

```diff
@@ -11,7 +11,14 @@
     </div>
     <div id="errorArea"></div>
     <div class="text-center">
-        <a data-bind="attr: { href: '/Poll/Index/' + pollId() }">Poll Link</a>
+        <form class="form-inline">
+            <div class="input-group">
+                <input readonly="readonly" class="form-control" data-bind="value: window.location.origin + '/Poll/Index/' + pollId(), click: function(i, e){ e.target.select() }" />
+                <span class="input-group-btn">
+                    <button class="btn btn-primary" type="button" data-bind="click: function(){window.location=('/Poll/Index/' + pollId())}">Go</button>
```

Comment:
Rather than a click, can we make the button have a href?

My first instinct was to right-click > Open in new tab, but I can't do that with the click function.